src/components/styles/_dashed.less

```diff
@@ -35,10 +35,34 @@
 
   &.black-btn {
     .borderRule(@black);
+
+    [data-theme="dark"] & {
+      .borderRule(@white);
+    }
+
+    @media (prefers-color-scheme: dark) {
+      .borderRule(@white);
+    }
+
+    &.dark-mode {
+      .borderRule(@white);
+    }
   }
 
   &.white-btn {
     border: dashed 2px @whiteBorder;
     color: @darkText !important;
+
+    [data-theme="dark"] & {
+      color: @white !important;
```

Comment:
Can you please put this line in a new mixin and call it from each? That way if we need to make any changes it will all be in one place. You can put the mixin inside the button.

Also, instead of putting rules separately for `[data-theme="dark"] &` and `&.dark-mode` I think you can put them in the same block of rules. For example:
```
[data-theme="dark"] &,
&.dark-mode {
...
}
```

Please do that in both button colors as it reduces repeated code

Reply:
ok

Reply:
Instead of creating a new mixin, can I use the **borderRule** mixing with white color, like this:

```less
&.white-btn {
    border: dashed 2px @whiteBorder;
    color: @darkText !important;

    [data-theme="dark"] &,
    &.dark-mode {
      .borderRule(@white);
    }

    @media (prefers-color-scheme: dark) {
      .borderRule(@white);
    }
  }
```

Reply:
That's fine as well